packages/beep/package.json

```diff
@@ -39,10 +39,13 @@
     "rollup": "^1.20.0||^2.0.0"
   },
   "devDependencies": {
-    "rollup": "^2.0.0",
-    "strip-ansi": "^5.2.0"
+    "rollup": "^2.23.0",
+    "strip-ansi": "^6.0.0"
   },
   "ava": {
+    "babel": {
+      "compileEnhancements": false
```

Comment:
can we document (or mention here) why this change was needed now and not previously?

Reply:
I believe this is due to how Ava 3 handles ESM. Not sure the `compileEnhancements` are strictly necessary, though, I was copying this from other places, but according to the docs it needed to be moved into the Babel option. There is a lot of plugins that need this now, you can easily test it: If you remove the Babel option entirely, the tests will fail. Should I document it for each plugin? Otherwise, I would rather hope that people look into the Ava docs when in doubt.

Reply:
Nah. I'm familiar with the option and suspected as much but wanted to get your take. Your comment here is good enough.